tests/test_methods/test_all_custom_interfaces.py

```diff
@@ -137,7 +150,7 @@ def test_adapter_loading(self):
         {
             "config_class": model_config["config_class"],
             "config": staticmethod(
-                lambda params=model_config["config_params"]: model_config["config_class"](**params)
+                lambda params=model_config["config_params"], cls=model_config["config_class"]: cls(**params)
```

Comment:
@lenglaender There was this sneaky bug that caused the whole test to actually only generate tests for the very last config in `MODEL_CONFIGS`. Thus before, we didn't actually test ModernBert at all.

Now, I needed to add `"reference_compile": False` to get it working at all, maybe we should check that again

Reply:
Oh, this is a really sneaky bug. As for the `"reference_compile": False'`: for me locally, all tests pass, even if it is not there (or explicitly set to true)?

Reply:
Oh that's a really sneaky bug! As for the `"reference_compile": False`: For me locally, all tests pass even if `reference_compile` is set to True or left out. Is this issue also prevalent in the GitHub test run or only locally on your computer?